lib/src/main/java/co/nimblehq/recentapps/thumbnailhiding/RecentAppsThumbnailHidingLifecycleTracker.kt

```diff
@@ -56,6 +83,14 @@ class RecentAppsThumbnailHidingLifecycleTracker : Application.ActivityLifecycleC
     }
 
     override fun onActivityDestroyed(activity: Activity) {
+        if (NavigationBarObserver.isAvailable() && navigationBarObserver?.activityContext == activity) {
+            navigationBarObserver?.run {
+                unregister()
+                removeOnNavigationBarListener(navigationBarListener)
+            }
+            navigationBarObserver = null
+            navigationBarListener = null
```

Comment:
Can we extract this to a separate method so that the code is not duplicated [here](https://github.com/nimblehq/recent-apps-thumbnail-hiding/pull/7/files#diff-20c2e5c3b31d2a91400a9e2d343b5f84075de981c6057b79c358a6701e509340R20-R25)?

Reply:
Oops, now I see your comment. Let me do it in the following PR

Reply:
@AVI5HEK fixed https://github.com/nimblehq/recent-apps-thumbnail-hiding/pull/6/commits/4063b62b09b91769c432bc950d2843fa7af14a3c